src/app/plot.py

```diff
@@ -0,0 +1,221 @@
+        width=575,
+        height=180
+    ).interactive()
+    
+    return chart.to_html()
+
+
+def plot_21(value):
+    
+    education_order = ["Less than bachelor's degree", "Bachelor's degree", 
+                       "Master's degree", "Doctoral degree"]
+    
+    if value == "World" or value is None:
+        country = data
+    else:
+        country = data.query("Country == @value")
+        
+    for idx, i in enumerate(country["FormalEducation"]):
+        if i in education_order[1:]:
+            continue
+        else:
+            print("Change")
+            country["FormalEducation"].iloc[idx] = "Less than bachelor's degree"
+
+    chart = alt.Chart(country).mark_bar().encode(
+            x=alt.X("Salary_USD", bin=alt.Bin(maxbins=20), title="Salary in USD"),
+            y=alt.Y("count()", title="Counts"),
+            color=alt.Color("FormalEducation", sort=education_order,
+            title="Education level"),
+            order=alt.Order('education_order:Q')
+        ).configure_legend(
+            orient='bottom',
+            titleFontSize=8,
+            labelFontSize=8
+        ).properties(
+            title = "Histogram of selected country",
+            width=350,
+            height=180,
+        ).configure_axis(
+            labelFontSize=12
+        )
+
+    return chart.to_html()
+
+
+def plot_11(xmax):
+    
+    world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
+    world["name"] = world["name"].apply(lambda x:str.lower(" ".join(x.split(" ")[0:2])))
+    
+    world = world.loc[world["name"] != "antarctica"]
+    source = data[["Country", "Salary_USD"]].groupby("Country").median().reset_index()
+    source["Country"] = source["Country"].apply(lambda x:str.lower(x))
+    source.rename({"Country":"name"}, axis=1, inplace=True)
+
+    datamap = pd.merge(world, source, how='left')
+    datamap['Salary_USD'] = datamap['Salary_USD'].fillna(0)
+    
+    chart = alt.Chart(datamap).mark_geoshape().encode( 
+        color=alt.Color(field = "Salary_USD",type = "quantitative",
+                        scale=alt.Scale(type = "sqrt"),
+                        legend=alt.Legend(title="Salary in USD",labelFontSize = 10,symbolSize = 10,titleFontSize=10)),
+        tooltip=['name:N', 'Salary_USD:Q', "alpha:Q"]
+    )
+    
+    if xmax is not None:
+        datamap["alpha"] = 1
+        datamap.loc[datamap["name"] == xmax.lower(), "alpha"] = 100
+        chart = chart.encode(
+            opacity=alt.Opacity(field = "alpha",type = "quantitative", scale=alt.Scale(type = "sqrt")),
+        )
+        
+    chart = chart.properties(
+        title='Median Salary of The World',
+        projection={"type":'mercator'},
+        width=600,
+        height=525,
+    ).configure_axis(
+        labelFontSize=10
+    )    
+    
+    return chart.to_html()
+
+def plot_13(DS_identity=['Yes', 'No', 'Sort of (Explain more)'], df=data.copy()):
+    # Clean data
+    df = df.dropna()
```

Comment:
Since this plot considers the `Salary` and `Coding Experience` only, it might be better to just drop rows if either of these columns are `NA` as opposed to dropping rows which have `NA` in any column

Reply:
@artanzand

Reply:
Actually, This one I can help change.